subprojects/dependency-management/src/test/groovy/org/gradle/internal/component/model/LocalComponentDependencyMetadataTest.groovy

```diff
@@ -100,11 +101,13 @@ class LocalComponentDependencyMetadataTest extends Specification {
             getName() >> 'foo'
             getAttributes() >> attributes(key: 'something')
             isCanBeConsumed() >> true
+            getCapabilities() >> Stub(CapabilitiesMetadataInternal)
```

Comment:
What's causing this to be called now?

Reply:
It was always called, but I added a cast [here](https://github.com/gradle/gradle/pull/23711/files#diff-047d2485129d198eab25159eaf4a303b7289d1235178f6c6d463a90bc1fdd0e2R205) which needs it to be the internal type, otherwise `ClassCastException`.  It needs the cast because I moved `isShadowedCapabilityOnly` to the internal type, so it doesn't clutter the public API.